docs/diagrams/buildkite_sync_sequence_diagram.puml

```diff
@@ -0,0 +1,59 @@
+@startuml buildkite_sequence_diagram
+    skinparam ParticipantPadding 20
+    skinparam BoxPadding 10
+
+    actor Bob
+
+    box Github
+        participant Repository
+    endbox
+
+    box Buildkite Awesome Pipeline
+        participant "Awesome pipeline"
+        participant Tests
+        Participant "Build and deploy"
+        Participant "Trigger sync pipeline"
```

Comment:
I'm not sure it should be different participants as these steps are performed inside the `Awesome Pipeline`